awx/ui_next/src/components/JobList/JobListCancelButton.jsx

```diff
@@ -62,7 +89,9 @@ function JobListCancelButton({ i18n, jobsToCancel, onCancel }) {
   };
 
   const isDisabled =
-    jobsToCancel.length === 0 || jobsToCancel.some(cannotCancel);
+    jobsToCancel.length === 0 ||
+    jobsToCancel.some(cannotCancelBecausePermissions) ||
+    jobsToCancel.some(cannotCancelBecauseNotRunning);
 
   const cancelJobText = i18n._(
     '{zeroOrOneJobSelected, plural, one {Cancel job} other {Cancel jobs}}',
```

Comment:
(This isn't directly related to the changes to this PR, but something I noticed).

We show `Cancel job(s)` for one action button and just `Delete` for the other. In my opinion, we don't need explicitly name the resource ('job(s)') and leaving it as `Cancel` would be more cohesive.

Reply:
> edit: Then again, maybe that could be ambiguous? Since sometimes a Cancel button means to close out of something e.g a form or modal.

Yea I did the exact same thing.  At first I made it `Cancel` to be consistent but thin I was like wait.... that's kind of weird.  Ended up going with Cancel job/jobs.  It's a weird case and I'm happy to change it if we can come up with something better.  cc @trahman73 if you're around and have any ideas.

Reply:
In the old UI, cancel used to be an icon that had a tooltip that said "Cancel job" I believe? Now that we are using buttons, keeping the "Cancel job" text feels right to me as just having "Cancel" follows our pattern of leaving an editable area.